bin/vcf_collect.py

```diff
@@ -0,0 +1,390 @@
+        metavar="GTF",
+        type=Path,
+        help="FusionInspector GTF output.",
+    )
+    parser.add_argument(
+        "--hgnc",
+        metavar="HGNC",
+        type=Path,
+        help="HGNC database.",
+    )
+    parser.add_argument("--sample", metavar="SAMPLE", type=Path, help="Sample name.", default="Sample")
+    parser.add_argument(
+        "--out",
+        metavar="OUT",
+        type=Path,
+        help="VCF output path.",
+    )
+    return parser.parse_args(argv)
+
+
+def header_def(sample):
+    """
+    Define the header of the VCF file
+    """
+    return '##fileformat=VCFv4.1\n\
+##ALT=<ID=BND,Description="Break end">\n\
+##INFO=<ID=SVTYPE,Number=1,Type=String,Description="Type of structural variant">\n\
+##INFO=<ID=CHRA,Number=1,Type=String,Description="Chromosome A">\n\
+##INFO=<ID=CHRB,Number=1,Type=String,Description="Chromosome B">\n\
+##INFO=<ID=GENEA,Number=.,Type=String,Description="Gene A">\n\
+##INFO=<ID=GENEB,Number=.,Type=String,Description="Gene B">\n\
+##INFO=<ID=POSA,Number=.,Type=String,Description="Breakpoint position A">\n\
+##INFO=<ID=POSB,Number=.,Type=String,Description="Breakpoint position B">\n\
+##INFO=<ID=ORIENTATION,Number=.,Type=String,Description="Strand1 and strand2 directions">\n\
+##INFO=<ID=FOUND_DB,Number=.,Type=String,Description="Databases in which the fusion has been found">\n\
+##INFO=<ID=FOUND_IN,Number=.,Type=String,Description="Callers that have found the fusion">\n\
+##INFO=<ID=TOOL_HITS,Number=.,Type=Integer,Description="Number of tools that found the fusion">\n\
+##INFO=<ID=SCORE,Number=.,Type=Float,Description="Score from fusionreport">\n\
+##INFO=<ID=FRAME_STATUS,Number=.,Type=Float,Description="Frame status of the fusion">\n\
+##INFO=<ID=TRANSCRIPT_ID_A,Number=.,Type=Float,Description="Transcript id A ">\n\
+##INFO=<ID=TRANSCRIPT_ID_B,Number=.,Type=Float,Description="Transcript id B">\n\
+##INFO=<ID=TRANSCRIPT_VERSION_A,Number=.,Type=Float,Description="Transcript version A">\n\
+##INFO=<ID=TRANSCRIPT_VERSION_B,Number=.,Type=Float,Description="Transcript version B">\n\
+##INFO=<ID=HGNC_ID_A,Number=.,Type=Float,Description="HGNC id A">\n\
+##INFO=<ID=HGNC_ID_B,Number=.,Type=Float,Description="HGNC id A">\n\
+##INFO=<ID=EXON_NUMBER_A,Number=.,Type=Float,Description="Exon number A">\n\
+##INFO=<ID=EXON_NUMBER_B,Number=.,Type=Float,Description="Exon number B">\n\
+##INFO=<ID=ANNOTATIONS,Number=.,Type=Float,Description="Annotations from FusionInspector">\n\
+##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">\n\
+##FORMAT=<ID=DV,Number=1,Type=Integer,Description="Number of paired-ends that support the event">\n\
+##FORMAT=<ID=RV,Number=1,Type=Integer,Description="Number of split reads that support the event">\n\
+##FORMAT=<ID=FFPM,Number=1,Type=Float,Description="Fusion fragments per million total RNA-seq fragments">\n\
+#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\t{}'.format(
+        sample
+    )
+
+
+def build_fusioninspector_dataframe(file):
```

Comment:
Nice to add type hints for readability 
```suggestion
def build_fusioninspector_dataframe(file: str) -> pd.DataFrame:
```